core/config/locales/en.yml

```diff
@@ -609,6 +609,7 @@ en:
       other: 'Subtotal (%{count} items)'
     carton_external_number: External Number
     carton_orders: 'Other orders with Carton'
+    carrier: Carrier
```

Comment:
Please, let's us start to keep the Rails way and use the attribute translations of the ShippingMethod model.